personal/Adam/Reports/zajecia2.Rmd

```diff
@@ -27,12 +29,12 @@ Który kraj posiada najmniejszą liczbę ludności i ile wynosi?
 
 ```{r pressure, echo=FALSE}
 min<-min(world$pop, na.rm=TRUE)
-wo<-world[world$pop==min,"name_long",na.rm=TRUE]
-plot(wo)
+popul<-world%>%filter(pop==min)%>%select(name_long,pop)

```

Comment:
lepsza nazwa, chociaż sugerująca, że to zbiór całej liczby ludności, a nie najmniej ludny kraj. Kiedy używasz pipe'ów przerzucaj je do następnej linijki dla lepszej czytelności kodu:
popul <- world %>%
    filter(pop==min)%>%
    select(name_long, pop)